twofactor/twofactor.go

```diff
@@ -336,17 +348,29 @@ func (tf *Twofactor) Auth(electionID []byte, authToken *uuid.UUID, authData []st
 		log.Warnf("verify challenge %s failed: %v", solution, err)
 		return AuthResponse{Error: "challenge not completed:" + err.Error()}
 	}
+	r, err := tf.NewBlindRequestKey()
+	if err != nil {
+		return AuthResponse{Error: "error getting new blind token:" + err.Error()}
+	}
+	tokenR := r.BytesUncompressed()
 
 	log.Infof("new user registered, challenge resolved %s", authData[0])
 	return AuthResponse{
 		Response: []string{"challenge resolved"},
 		Success:  true,
-		TokenR:   tf.NewRequestKey(),
+		TokenR:   tokenR,
 	}
 }
 
 func (tf *Twofactor) Sign(electionID, payload, tokenR internal.HexBytes, address string) AuthResponse {
-	signature, err := tf.SignECDSA(tokenR, payload, electionID)
+	r, err := blindsecp256k1.NewPointFromBytesUncompressed(tokenR)
+	if err != nil {
+		return AuthResponse{
+			Success: false,
+			Error:   err.Error(),
+		}
+	}
```

Comment:
Same here, I think that the bytes-to-point conversion can be included into the `SignBlind` method.